### PR TITLE
Fix refresh_user_list

### PR DESCRIPTION
**Pull request by acrogenesis**
When adding or removing users to a project the list didn't refresh correctly, this fixes the problem.

### CODE REVIEW

**Comment by acrogenesis** [comment]
I'm not sure why the tests are failing
```
Projects
  when logged in
    as non-admin
      join project
        joins a project (FAILED - 1)
```

**Comment by eddiecode** [comment]
Hey, @acrogenesis thanks for your contribution.

Sometimes the tests break, try to run the tests again.

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@ def index
   end
 
   def create
+    @current_team_users = current_team.users.where.not(id: @project.users).order(:name)
```
Comment (talyssonoc) [inline app/controllers/users_controller.rb]: You could extract it to a method, like:

```ruby
def create
  @current_team_users = current_team_users
end

def destroy
  # ...
  @current_team_users = current_team_users
end

# ...

def current_team_users
  current_team.users.where.not(id: @project.users).order(:name)
end
```

**Comment by acrogenesis** [comment]
@talyssonoc ready

**Comment by andrerpbts** [comment]
Can you add a CHANGELOG entry, please?

**Comment by acrogenesis** [comment]
@andrerpbts done!

**Pull request merged**